src/docs/product/accounts/sso/azure-sso.mdx

```diff
@@ -76,7 +76,9 @@ description: "Set up Azure Active Directory Single Sign-On (SSO) on Sentry."
 
 <Note>
 
-This feature is not currently available to all Sentry users. If you are interested in participating in SCIM's beta, please reach out to [support@sentry.io](mailto:support@sentry.io).
+Azure SCIM integration is not currently available to all Sentry users. If you are interested in participating in SCIM's beta, please reach out to [support@sentry.io](mailto:support@sentry.io).
+
+Also, this feature is only available if your organization is on a Business or Enterprise plan; it is not available on Trial plans.
```

Comment:
@JoshFerge - I wanted to make the information about SCIM requiring certain plans more prominent, but this section of the Azure page already has a note. Are you good with this treatment?

Reply:
sounds good!